### PR TITLE
Prac 06 feedback

### PR DESCRIPTION
**Pull request by ivy-naw-mth**
@ChristineYamin @Anastasia-Eremenko
Would you guys mind checking my code?
Thanks in advance! ^^

### CODE REVIEW

File: prac_06/car.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        Drive given distance if car has enough fuel
+        or drive until fuel runs out return the distance actually driven.
+        """
+        if distance > self.fuel:
+            distance = self.fuel
+            self.fuel = 0
+        else:
+            self.fuel -= distance
+        self._odometer += distance
+        return distance
```
Comment (ChristineYamin) [inline prac_06/car.py:33]: correct coding

File: prac_06/languages.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+from prac_06.programming_language import ProgrammingLanguage
+
+python = ProgrammingLanguage("Python", "Dynamic", True, 1991)
+ruby = ProgrammingLanguage("Ruby", "Dynamic", True, 1995)
+visual_basic = ProgrammingLanguage("Visual Basic", "Static", False, 1991)
+print(python)
```
Comment (ChristineYamin) [inline prac_06/languages.py:6]: You have to print ruby and visual_basic

File: prac_06/guitars.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        guitars.append(Guitar(name, year, cost))
+        print(f"{Guitar(name, year, cost)} added.", end="\n\n")
+        name = input("Name: ")
+
+    print("\n...snip...\n")
+    print("These are my guitars:")
+    guitars.append(Guitar("Gibson L-5 CES", 1922, 16035.40))
+    guitars.append(Guitar("Line 6 JTV-59", 2010, 1512.9))
+
+    if guitars:
```
Comment (ChristineYamin) [inline prac_06/guitars.py:21]: You can use the method that the instruction gives us.
for i, guitar in enumerate(guitars, 1):
    vintage_string = " (vintage)" if guitar.is_vintage() else ""

Comment (ivy-naw-mth) [reply prac_06/guitars.py:21]: Got it, thanks for the comment! ^^

File: prac_06/languages.py (hunk 2 above)
Comment (ivy-naw-mth) [reply prac_06/languages.py:6]: Done fixing it!

File: prac_06/guitar.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+    def is_vintage(self):
+        """check if vintage or not"""
+        age = self.get_age()
+        if age > 50:
+            return True
+        else:
+            return False
+
+    def __lt__(self, other):
```
Comment (Anastasia-Eremenko) [inline prac_06/guitar.py:27]: i think you can put a docstring here :)
to describe what this function does
 def __lt__(self, other):

Comment (Anastasia-Eremenko) [reply prac_06/guitar.py:27]: also i think by convention miss Cynthia wants us to put docstrings on all functions

Comment (ivy-naw-mth) [reply prac_06/guitar.py:27]: I totally forgot about it. Thanks for your comment.

**Pull request merged**